float32 x
float32 y
sensor_msgs/CameraInfo camera_info

